samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -32,9 +32,11 @@ static struct bt_mesh_cfg_srv cfg_srv = {
 
 	.default_ttl = 7,
 
-	/* 3 transmissions with 20ms interval */
-	.net_transmit = BT_MESH_TRANSMIT(2, 20),
-	.relay_retransmit = BT_MESH_TRANSMIT(2, 20),
+	/* 5 transmissions with 30ms interval */
+	.net_transmit = BT_MESH_TRANSMIT(5, 30),
```

Comment:
What's the reasoning for this? Do you have any performance measurements to justify these changes?

Reply:
Kindly consider above explanation.